docs/docs/configuration.md

```diff
@@ -204,6 +204,23 @@ For example:
 # In this example, the following will not be packaged in to MTA archive:
 # all files with the"txt" extension and the "mtaignore" folder within the "myfolder" directory. 
      
+```
+From MBT 1.2.25 version, the `build-parameters ignore` attribute will be upgraded to support [full glob pattern](https://en.wikipedia.org/wiki/Glob_(programming)). Wildcards, such as `*`,`!`,`?` can be used to match characters. 
```

Comment:
Can we change it to "full glob patterns" instead of "full glob pattern" throughout?

Reply:
OK Done